### DISCUSSION TITLE
I can't cancel transfers

### DISCUSSION THREAD

**Discussion by ECPerkins** [iterate-ch/cyberduck#15681] [Q&A]
When uploading or downloading files, the Stop button does nothing in the transfer window. I have to exit the program completely, and upon reopening it, delete the file from the transfer window before I reconnect to the server.

**Comment by dkocher** [comment]
I cannot seem to reproduce this. If you are transferring files using [multiple connections](https://docs.cyberduck.io/cyberduck/transfer/#limit-number-of-parallel-connections) it may take some time until the transfer is stopped.
